website/docs/reference/appsmith-framework/widget-actions/intervals-time-events.md

```diff
@@ -68,7 +68,7 @@ clearInterval("myTimer");
 ### Signature
 
 ```javascript
-setTimeout(callbackFunction: Function, delay: number, args?: any)
+setTimeout(callbackFunction: Function, delay: number)
```

Comment:
`setTimeout(callbackFunction: Function, delay = 0, param1, param2, /* … ,*/ paramN)` is the right syntax.
"params 1-N" are optional parameters.

Listing some ways to use setTimeout below.
`setTimeout(() => showAlert("Hello after 1 second "), 1000)`

`setTimeout((message) => showAlert(message), 1000,  "Hello after 1 second") //3rd argument of setTimeout function is passed as the first argument to callbackFunction`

`setTimeout(() => showAlert("Hello"))`

Reply:
Thank you for those examples, that was helpful especially with the one that passes parameters. Does the updated version look better?